patches/react-native-keychain+6.1.1.patch

```diff
@@ -130,3 +130,29 @@ index b1a11f2..6a055c2 100644
  /**
```

Comment:
In the keychain patch for index.js, lines 112 and lines 123, you have getAllInternetCredentials exported as a function of type: Promise<false | UserCredentials> (should be UserCredentials[] here too?). Also same update for getAllInternetCredentailsKeys (with the second option as String[])?

Reply:
Good catch. It was fine in the .d.ts file but wrong in the signature. It should be `AllCredentials` which is basically `UserCredentials[]`